internal/services/storage/storage_container_resource.go

```diff
@@ -93,8 +98,9 @@ func expandStorageContainerAccessLevel(input string) containers.AccessLevel {
 	return containers.AccessLevel(input)
 }
 
-func storageContainerCreate(d *pluginsdk.ResourceData, meta interface{}) error {
+func resourceStorageContainerCreate(ctx context.Context, d *pluginsdk.ResourceData, meta interface{}) diag.Diagnostics {
 	storageClient := meta.(*clients.Client).Storage
+	_ = ctx
```

Comment:
Remove this and return the CRUD functions to this for now `storageContainerCreate(d *pluginsdk.ResourceData, meta interface{}) error`

Reply:
ready!